DESCRIPTION

```diff
@@ -47,7 +47,7 @@ Imports:
     rmarkdown, 
     shiny
 Remotes: 
-    RMI-PACTA/pacta.executive.summary, 
+    RMI-PACTA/pacta.executive.summary@update-sbti-section-scorecard, 
```

Comment:
I don't think we should be targeting a PR branch on main?

Reply:
this is for testing purposes I believe. And it is not going to be merged into main

Reply:
ok, then can we turn it back into a draft, mark it clearly as such in the title, and not request reviews from people?

Reply:
@cjyetman This PR isn't targeting `main`, it's targeting `update-executive-summary` which is itself is a PR branch. 

This whole PR is just a glorified PR commit